tests/openvino/native/common.py

```diff
@@ -26,6 +26,10 @@ def compare_nncf_graphs(model: ov.Model, path_ref_graph: str) -> None:
     compare_nx_graph_with_reference(nx_graph, path_ref_graph, check_edge_attrs=True, unstable_node_names=True)
 
 
+def dump_model(model: ov.Model, xml_path: str, bin_path: str):
+    ov.serialize(model, xml_path, bin_path)
```

Comment:
Do you really need a new function for this?

Reply:
Why not? Don't need to remember the method from `ov` to use